modin/data_management/factories/dispatcher.py

```diff
@@ -11,20 +11,47 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""
+Contain IO dispatcher class.
+
+Dispatcher routes the work to excecution engine-specific functions.
+"""
+
 from modin.config import Engine, Backend, IsExperimental
 from modin.data_management.factories import factories
 from modin.utils import get_current_backend
 
 
 class FactoryNotFoundError(AttributeError):
+    """
+    ``FactoryNotFound`` exception class.
+
+    Raise when no matching factory could be found.
+    """
+
     pass
 
 
 class StubIoEngine(object):
+    """
+    IO-Engine that does nothing more than raise NotImplementedError when any method is called.
+
+    Parameters
+    ----------
+    factory_name : str
+        Factory name, which will be reflected in error messages.
+
+    Notes
+    -----
+    Used for testing purposes.
+    """
+
     def __init__(self, factory_name=""):
         self.factory_name = factory_name or "Unknown"
 
-    def __getattr__(self, name):
+    def __getattr__(self, name):  # noqa: PR01, RT01
```

Comment:
`PR01: parameter 'name' not documented; RT01: missing 'return' section`.
This method is trivial, both this sections is described in the summary docstring

Reply:
I would rather we follow formal template, shouldn't be too hard.